Cargo.toml

```diff
@@ -101,8 +101,7 @@ paste = "1.0.15"
 prettyplease = "0.2.17"
 proc-macro2 = "1.0"
 quote = "1.0.37"
-rand = "0.8.5"
-rand_chacha = "0.3.1"
+rand = { version = "0.8", features = ["std"] }
```

Comment:
localize the features you need to the crate that needs them

but `std` is already a default feature, so this is not needed